expertise/service/celery_tasks.py

```diff
@@ -23,43 +26,68 @@ def update_status(config, new_status, desc=None):
     else:
         config.description = desc
     config.mdate = int(time.time() * 1000)
-    config.save()
+    redis_db = RedisDatabase(connection_pool=redis_conn_pool)
+    redis_db.save_job(config)
+
+def on_failure_userpaper(self, exc, task_id, args, kwargs, einfo):
+    config, logger = args[0], args[2]
```

Comment:
could you import the logger here instead of passing it as an argunment?

Reply:
I tried directly to import the Flask current app logger but it seems like I would have to make additional changes elsewhere to make Celery aware of Flask's current app and I'm not sure if that would be worth it